frameworks/digital-outcomes-and-specialists/questions/briefs/evaluationTypeOutcomes.yml

```diff
@@ -2,20 +2,20 @@ name: Assessment methods
 id: evaluationType
 question: How you’ll assess suppliers
 question_advice: |
-  Choose how you want to assess suppliers.
-  
-  You can only use the methods you choose here.
-  
-  You must ask for a written proposal.
-  
+  All suppliers that reach the assessment stage will be asked to provide a written proposal.
```

Comment:
The wording here (and for other lots) implies that a written proposal/work history will be asked for automatically by the system.  Is that true? (I don't think so?)

Would something like `You must ask all suppliers that reach the assessment stage for a written proposal.` be better, to make it clear that it's something they have to do, not something we will do for them.